### DISCUSSION TITLE
Non latin letters are not accepted in a search filter

### DISCUSSION THREAD

**Discussion by boo-code** [directus/directus#12724] [Archive]
Non Latin letters in the search filter do not work. No results are returned.

![image](https://user-images.githubusercontent.com/65190196/162986039-61c66b6e-e049-4b03-b8d7-af5e75f57eb7.png)

### To Reproduce

1. Add a record with ą,č,ę,ė,į,š,ų,ū,ž or any other non-Latin letter. 
2. Use a search field filter in the collection view to find the record with such a letter.

<details>
<summary>More info</summary>

### What version of Directus are you using?

9.8.0

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres 14

### What browser are you using?

Any

### How are you deploying Directus?

Docker

</details>

**Comment by azrikahar** [comment]
I can't seem to reproduce this on my end, and I've tested both SQLite and Postgres as well:

https://user-images.githubusercontent.com/42867097/163108092-9ebcf1dd-3cce-4ef2-8667-75aac837eba4.mp4

Would you be able to test this with another database? just so we can be sure whether this only happen to this database specifically as I'm not sure how to best replicate this issue locally.

**Comment by boo-code** [comment]
Sorry, my fault. It's not the letters itself. The filter is case-sensitive. I had a word starting with Š and entered š in a filter. Then I got no results returned. So, actual problem is that the search field is case-insensitive, and the filter is case-sensitive. It creates confusion.

This issue should be changed to "case-sensitive filter confusion".

More letters for you to test case-sensitivity: Ą,Č,Ę,Ė,Į,Š,Ų,Ū,Ž.

**Comment by azrikahar** [comment]
(Moving this to discussion as it's more of a feature request than an unintended bug)

---

Oh gotcha, no worries! That is indeed the current limitation which has been discussed in #2981. The prevalent roadblock was adding support for every database vendors. Currently you can use [search](https://docs.directus.io/reference/query/#search) which is case-insensitive. Searching directly in the search input (without filters) will work since it uses the `search` query parameter as seen in the screenshot below 👍

![chrome_pH2VkR42cK](https://user-images.githubusercontent.com/42867097/163153275-1c7fbd46-8894-47aa-92de-ab04616cc531.png)

On that note, the good news is support for case-insensitive filters is being added in #11737! So feel free to subscribe to it to get notified when it's merged.
